package.json

```diff
@@ -75,11 +75,13 @@
     "graphql": "^16.0.0",
     "history": "^4.6.3",
     "hoist-non-react-statics": "^3.3.0",
+    "inactivity-timer": "^1.0.0",
```

Comment:
uses the MIT license but I think that's compatible with Apache v2.  Also the last update was 4 years ago... any concerns about either of these points?